### PR TITLE
Closes #288: editmode INFO/FILE removal

### LINKED ISSUE #288: Check if DatasetPage.EditMode INFO and FILE are used in application - if not then remove them

### PR DESCRIPTION
**Pull request by dbojanek**

### CODE REVIEW

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java [hunk 1]
```diff
@@ -1274,9 +1274,9 @@ public boolean isSameTermsOfUse(FileTermsOfUse termsOfUse1, FileTermsOfUse terms
     public List<DataFile> createDataFiles(DatasetVersion version, InputStream inputStream, String fileName, String suppliedContentType) throws IOException {
         List<DataFile> datafiles = new ArrayList<>();
 
-        if (true) {
-            throw new RuntimeException();
-        }
+//        if (true) {
```
Comment (MrK191) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java]: If it's commented it should be removed.

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 2]
```diff
@@ -2228,20 +2218,12 @@ private void populateDatasetUpdateFailureMessage() {
...
             }
         } else {
-
-            if (editMode.equals(EditMode.CREATE)) {
-                JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.createFailure"));
-            }
-            if (editMode.equals(EditMode.FILE)) {
-                JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.filesFailure"));
-            }
+            JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.createFailure"));
```
Comment (MrK191) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java:2225]: I understand that edit mode in this place is always in create mode?

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 3]
```diff
@@ -139,8 +139,7 @@
     private static final Logger logger = Logger.getLogger(DatasetPage.class.getCanonicalName());
 
     public enum EditMode {
-
-        CREATE, INFO, FILE
+        CREATE
```
Comment (MrK191) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java:142]: This will be removed in another task?

**Review by MrK191 — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java (hunk 2 above)
Comment (dbojanek) [reply dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java:2225]: Yes, what's left is editMode == null and editMode == 'CREATE'/

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java (hunk 3 above)
Comment (dbojanek) [reply dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java:142]: Probably, when we separate creating datasets from main dataset page.

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java (hunk 1 above)
Comment (dbojanek) [reply dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java]: Oops, forgot about it.

**Review by MrK191 — APPROVED** [review]

**Pull request merged**